docs/contributing/release.rst

```diff
@@ -11,37 +11,33 @@ Packages in this repository
 
 This repo has more than one package.
 
-#. The primary ``jupyter-server-proxy`` python package. This contains the actual proxying
-   code, and the classic notebook extension. This is published on `PyPI <https://pypi.org>`_.
+#. The primary ``jupyter-server-proxy`` python package. This contains the actual
+   proxying code, and the classic notebook extension. This is published on `PyPI
+   <https://pypi.org/project/jupyter-server-proxy>`_.
 
-#. The JupyterLab plugin, ``jupyterlab-server-proxy``. This provides the launcher icons in
-   JupyterLab.
+#. The JupyterLab plugin, ``jupyterlab-server-proxy``. This provides the
+   launcher icons in JupyterLab. This is packaged into the Python package but
+   also published to `NPM
+   <https://www.npmjs.com/package/@jupyterlab/server-proxy>`_ for JupyterLab 2
+   compatibility.
 
-#. Various python packages in ``contrib/``. These are convenience packages that help set up
-   a particular application for use - such as RStudio or Theia.
+#. Various python packages in ``contrib/``. These are convenience packages that
+   help set up a particular application for use - such as Theia.
```

Comment:
Maybe we should move these to a new repo, `jupyter-server-proxy-contrib`?

Reply:
Maybe, a decision I lack knowledge to weigh in properly on. Will you open an issue with the question/suggestion?

Reply:
https://github.com/jupyterhub/jupyter-server-proxy/issues/251